.github/workflows/agw-workflow.yml

```diff
@@ -12,6 +12,10 @@ on:  # yamllint disable-line rule:truthy
       - v1.*
     types: [opened, reopened, synchronize]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
```

Comment:
what does the || mean in this context?

Reply:
it's the regular `or` .
The concurrency group will be the pr number if it's defined or the sha of the commit.

Reply:
ohhh i see - i guess pr number doesn't exist for master runs?

Reply:
yes exact!